src/System.Reflection.Metadata/specs/Ecma-335-Issues.md

```diff
@@ -0,0 +1,133 @@
+# Known Issues in ECMA-335 CLI Specification 
```

Comment:
I'd distinguish between ECMA-335 spec issues and implementation issues. Implementation issues in CLR should certainly be tracked by a document in coreclr repo. Since the goal of System.Reflection.Metadata package is provide public API for reading and writing ECMA-335 compliant files I think this repo is the right place for the spec errata.